docs/OTLabels/validation/modelvalidation.md

```diff
@@ -0,0 +1,234 @@
+| ![Scene 3 TIDE](assets/plots/scene_3/tide_errors_diagram_scene_3.png)  | ![Scene 3 TIDE Night](assets/plots/scene_3/tide_errors_diagram_scene_3_night.png)          |
+| ![Scene 3 TIDE Stacked](assets/plots/scene_3/tide_stacked_diagram_scene_3.png)  | ![Scene 3 TIDE Stacked Night](assets/plots/scene_3/tide_stacked_diagram_scene_3_night.png) |
+
+Comparing these two datasets, it is not surprising that the models' mAP evaluated on the dataset taken at daylight fares much better to the one evaluated at nighttime.
+The low light condition makes it really hard for the models to detect any of the objects.
+The biggest drop in performance in terms of the mAP metric is seen from the YOLOv5s model.
+
+But then, why is the **Missed Ground Truth Error** (Miss) much lower and the **Classification Error** (Cls) that much higher of the nighttime dataset of Scene 3 compared to the one taken at daylight?
+Let's have a closer look at the two datasets first:
+
+| Scene 3 at daylight                                                      | Scene 3 at nighttime                                       |
+|--------------------------------------------------------------------------|------------------------------------------------------------|
+| ![Scene 3 Ground Truth Information](assets/plots/scene_3/gt_scene_3.svg) | ![Scene 3 Night](assets/plots/scene_3/gt_scene_3_night.svg)|
+
+Overall, the amount of ground truth bounding boxes in scene 3 taken at daylight is more than twice as big as the one taken at nighttime.
+This implies that there is less traffic at nighttime, which makes sense.
+Thus, there is not much to detect at night resulting in a lower **Missed Ground Truth Error**.
+On the other hand, the classification error for all models is above 40% with the exception of the YOLOv5l model whose classification error is at approximately 24%.
+In addition to that the **Localization Error** (Loc) is very low.
+Meaning, the model is able to detect objects very well, but has trouble assigning them to the correct class in badly lit areas.
+Here is an example where a car is barely recognizable:
+
+![Scene 3 Night Badly Lit Car](assets/scenes/scene_3_night_badly_lit_car.jpg)
+
+Speaking of scene 3 at daylight, the **MissedGround Truth Error** is relatively high across all models which is due to the same reason as explained in **Scene 1**.
+Traffic objects whose bounding boxes are really small might be detected by the models resulting in misses.
+
+But it is also important to take into account that there is the possibility of not moving objects appearing in every image at the same spot in the dataset.
+Such case can be seen in scene 3 at nighttime where cars have been parked on the sidewalk in the following image:
+
+![Scene 3 Night Parked Cars](assets/scenes/scene_3_night_parked_cars.jpg)
+
+Thus, such case could influence the values of the calculated metrics for better or worse.
+
+#### Scene 4
```

Comment:
Contrary to the header, scene 5 images and plots appear in the tables (and are not mentioned in the text). Why is that?

Reply:
Thank you Martin~ They indeed shouldn't be there. I forgot to remove the scene 5 images and plots when refactoring them to their own separate chapter. I have removed them in my latest commits.